.circleci/comment.js

```diff
@@ -63,18 +63,23 @@ async function run() {
   } else {
     pr = process.env.CIRCLE_PULL_REQUEST.split('/').pop();
   }
-
+  console.log('PR number to comment on', pr);
   if (pr != null) {
-    await octokit.issues.createComment({
-      owner: 'adobe',
-      repo: 'react-spectrum',
-      issue_number: pr,
-      body: `Build successful! 🎉
+    try {
+      await octokit.issues.createComment({
+        owner: 'adobe',
+        repo: 'react-spectrum',
+        issue_number: pr,
+        body: `Build successful! 🎉
 
   * [View the storybook](https://reactspectrum.blob.core.windows.net/reactspectrum/${process.env.CIRCLE_SHA1}/storybook/index.html)
   * [View the storybook-17](https://reactspectrum.blob.core.windows.net/reactspectrum/${process.env.CIRCLE_SHA1}/storybook-17/index.html)
   * [View the storybook-16](https://reactspectrum.blob.core.windows.net/reactspectrum/${process.env.CIRCLE_SHA1}/storybook-16/index.html)
   * [View the documentation](https://reactspectrum.blob.core.windows.net/reactspectrum/${process.env.CIRCLE_SHA1}/docs/index.html)`
-    });
+      });
+    } catch (err) {
+
```

Comment:
console log the err?

Reply:
oh what da heck, I swear I had a console.error here